### PR TITLE
GA_GS FLMA-B Deploy CSV Updates

### PR DESCRIPTION
**Pull request by rtravis-whoi**
Updates per metadata review to:
GA03FLMA_Deploy.csv
GA03FLMB_Deploy.csv
GS03FLMA_Deploy.csv
GS03FLMB_Deploy.csv

Also updated CUIDs per recent alignment w/ R2R CUID formats

Used attached SSM depths spreadsheet for sensor depths.
[Subsurface_Mooring_Depth_Details_2020-07-10.xlsx](https://github.com/WHOIGit/ooicgsn-asset-management/files/4997933/Subsurface_Mooring_Depth_Details_2020-07-10.xlsx)

### CODE REVIEW

**Comment by rtravis-whoi** [comment]
[Critical Metadata Revew [DT-15]](https://trello.com/c/OuquTzsl/15-critical-metadata-revew-dt-15)

File: deployment/GA03FLMA_Deploy.csv [hunk 1]
```diff
@@ -1,55 +1,55 @@
...
+AT26-30,,NBP1510,,GA03FLMA-RIM01-02-CTDMOH049,1,1,2015-03-17T19:45:00,2015-11-23T11:46:00,CGMGA-03FLMA-00001,,CGINS-CTDMOH-12394,-42.493,-42.8705,,750,5165,
+AT26-30,,NBP1510,,GA03FLMA-RIM01-02-CTDMOH050,1,1,2015-03-17T19:45:00,2015-11-23T11:46:00,CGMGA-03FLMA-00001,,CGINS-CTDMOH-12618,-42.493,-42.8705,,1000,5165,
+AT26-30,,NBP1510,,GA03FLMA-RIM01-02-CTDMOH051,1,1,2015-03-17T19:45:00,2015-11-23T11:46:00,CGMGA-03FLMA-00001,,CGINS-CTDMOH-12619,-42.493,-42.8705,,1500,5165,
+AT26-30,,NBP1510,,GA03FLMA-RIS01-00-SIOENG000,1,1,2015-03-17T19:45:00,2015-11-22T12:08:00,CGMGA-03FLMA-00001,,CGCON-SIOCON-13989,-42.493,-42.8705,,30,5165,
+AT26-30,,NBP1510,,GA03FLMA-RIS01-03-DOSTAD000,1,1,2015-03-17T19:45:00,2015-11-22T12:08:00,CGMGA-03FLMA-00001,,CGINS-DOSTAD-00428,-42.493,-42.8705,,30,5165,
+AT26-30,,NBP1510,,GA03FLMA-RIS01-04-PHSENF000,1,1,2015-03-17T19:45:00,2015-11-22T12:08:00,CGMGA-03FLMA-00001,,CGINS-PHSENF-P0141,-42.493,-42.8705,,30,5165,
+AT26-30,,NBP1510,,GA03FLMA-RIS01-05-FLORTD000,1,1,2015-03-17T19:45:00,2015-11-22T12:08:00,CGMGA-03FLMA-00001,,CGINS-FLORTD-01215,-42.493,-42.8705,,30,5165,
```
Comment (snwhite) [inline deployment/GA03FLMA_Deploy.csv:19]: Deployment 1 – Looks good.

File: deployment/GA03FLMA_Deploy.csv [hunk 2]
```diff
@@ -1,55 +1,55 @@
...
+NBP1510,,NBP1609,,GA03FLMA-RIM01-02-CTDMOH049,2,1,2015-11-25T19:21:00,2016-11-07T09:49:00,CGMGA-03FLMA-00002,,CGINS-CTDMOH-13627,-42.49457,-42.92112,,750,5142,
+NBP1510,,NBP1609,,GA03FLMA-RIM01-02-CTDMOH050,2,1,2015-11-25T19:21:00,2016-11-07T09:49:00,CGMGA-03FLMA-00002,,CGINS-CTDMOH-13631,-42.49457,-42.92112,,1000,5142,
+NBP1510,,NBP1609,,GA03FLMA-RIM01-02-CTDMOH051,2,1,2015-11-25T19:21:00,2016-11-07T09:49:00,CGMGA-03FLMA-00002,,CGINS-CTDMOH-13632,-42.49457,-42.92112,,1500,5142,
+NBP1510,,NBP1609,,GA03FLMA-RIS01-00-SIOENG000,2,1,2015-11-25T19:21:00,2016-11-05T18:58:00,CGMGA-03FLMA-00002,,CGCON-SIOCON-14875,-42.49457,-42.92112,,30,5142,
+NBP1510,,NBP1609,,GA03FLMA-RIS01-03-DOSTAD000,2,1,2015-11-25T19:21:00,2016-11-05T18:58:00,CGMGA-03FLMA-00002,,CGINS-DOSTAD-00413,-42.49457,-42.92112,,30,5142,
+NBP1510,,NBP1609,,GA03FLMA-RIS01-04-PHSENF000,2,1,2015-11-25T19:21:00,2016-11-05T18:58:00,CGMGA-03FLMA-00002,,CGINS-PHSENF-P0181,-42.49457,-42.92112,,30,5142,
+NBP1510,,NBP1609,,GA03FLMA-RIS01-05-FLORTD000,2,1,2015-11-25T19:21:00,2016-11-05T18:58:00,CGMGA-03FLMA-00002,,CGINS-FLORTD-01300,-42.49457,-42.92112,,30,5142,
```
Comment (snwhite) [inline deployment/GA03FLMA_Deploy.csv:37]: Deployment 2 – Looks good.

File: deployment/GA03FLMA_Deploy.csv [hunk 3]
```diff
@@ -1,55 +1,55 @@
...
+NBP1609,,AT39-03,,GA03FLMA-RIM01-02-CTDMOH049,3,1,2016-10-30T23:17:00,2018-01-09T12:51:00,CGMGA-03FLMA-00003,,CGINS-CTDMOH-12395,-42.49212833,-42.8635,,750,5166,
+NBP1609,,AT39-03,,GA03FLMA-RIM01-02-CTDMOH050,3,1,2016-10-30T23:17:00,2018-01-09T12:51:00,CGMGA-03FLMA-00003,,CGINS-CTDMOH-12387,-42.49212833,-42.8635,,1000,5166,
+NBP1609,,AT39-03,,GA03FLMA-RIM01-02-CTDMOH051,3,1,2016-10-30T23:17:00,2018-01-09T12:51:00,CGMGA-03FLMA-00003,,CGINS-CTDMOH-12618,-42.49212833,-42.8635,,1500,5166,
+NBP1609,,AT39-03,,GA03FLMA-RIS01-00-SIOENG000,3,1,2016-10-30T23:17:00,2018-01-09T10:50:00,CGMGA-03FLMA-00003,,CGCON-SIOCON-13987,-42.49212833,-42.8635,,30,5166,
+NBP1609,,AT39-03,,GA03FLMA-RIS01-03-DOSTAD000,3,1,2016-10-30T23:17:00,2018-01-09T10:50:00,CGMGA-03FLMA-00003,,CGINS-DOSTAD-00226,-42.49212833,-42.8635,,30,5166,
+NBP1609,,AT39-03,,GA03FLMA-RIS01-04-PHSENF000,3,1,2016-10-30T23:17:00,2018-01-09T10:50:00,CGMGA-03FLMA-00003,,CGINS-PHSENF-P0147,-42.49212833,-42.8635,,30,5166,
+NBP1609,,AT39-03,,GA03FLMA-RIS01-05-FLORTD000,3,1,2016-10-30T23:17:00,2018-01-09T10:50:00,CGMGA-03FLMA-00003,,CGINS-FLORTD-01219,-42.49212833,-42.8635,,30,5166,
```
Comment (snwhite) [inline deployment/GA03FLMA_Deploy.csv:55]: Deployment 3 – Looks good.

File: deployment/GA03FLMB_Deploy.csv [hunk 4]
```diff
@@ -1,55 +1,55 @@
...
+NBP1510,,NBP1609,,GA03FLMB-RIM01-02-CTDMOH069,2,1,2015-11-21T21:51:00,2016-11-01T12:40:00,CGMGA-03FLMB-00002,,CGINS-CTDMOH-13647,-42.49243,-42.1626,,750,5093,
+NBP1510,,NBP1609,,GA03FLMB-RIM01-02-CTDMOH070,2,1,2015-11-21T21:51:00,2016-11-01T12:40:00,CGMGA-03FLMB-00002,,CGINS-CTDMOH-13634,-42.49243,-42.1626,,1000,5093,
+NBP1510,,NBP1609,,GA03FLMB-RIM01-02-CTDMOH071,2,1,2015-11-21T21:51:00,2016-11-01T12:40:00,CGMGA-03FLMB-00002,,CGINS-CTDMOH-13646,-42.49243,-42.1626,,1500,5093,
+NBP1510,,NBP1609,,GA03FLMB-RIS01-00-SIOENG000,2,1,2015-11-21T21:51:00,2016-11-01T10:10:00,CGMGA-03FLMB-00002,,CGCON-SIOCON-14882,-42.49243,-42.1626,,30,5093,
+NBP1510,,NBP1609,,GA03FLMB-RIS01-03-DOSTAD000,2,1,2015-11-21T21:51:00,2016-11-01T10:10:00,CGMGA-03FLMB-00002,,CGINS-DOSTAD-00467,-42.49243,-42.1626,,30,5093,
+NBP1510,,NBP1609,,GA03FLMB-RIS01-04-PHSENF000,2,1,2015-11-21T21:51:00,2016-11-01T10:10:00,CGMGA-03FLMB-00002,,CGINS-PHSENF-P0180,-42.49243,-42.1626,,30,5093,
+NBP1510,,NBP1609,,GA03FLMB-RIS01-05-FLORTD000,2,1,2015-11-21T21:51:00,2016-11-01T10:10:00,CGMGA-03FLMB-00002,,CGINS-FLORTD-01301,-42.49243,-42.1626,,30,5093,
```
Comment (snwhite) [inline deployment/GA03FLMB_Deploy.csv:37]: Deployment 2 – Looks good.

File: deployment/GA03FLMB_Deploy.csv [hunk 5]
```diff
@@ -1,55 +1,55 @@
...
+NBP1609,,AT39-03,,GA03FLMB-RIM01-02-CTDMOH069,3,1,2016-10-31T22:37:00,2018-01-12T13:07:00,CGMGA-03FLMB-00003,,CGINS-CTDMOH-12619,-42.49601667,-42.12545,,750,5165,
+NBP1609,,AT39-03,,GA03FLMB-RIM01-02-CTDMOH070,3,1,2016-10-31T22:37:00,2018-01-12T13:07:00,CGMGA-03FLMB-00003,,CGINS-CTDMOH-12388,-42.49601667,-42.12545,,1000,5165,
+NBP1609,,AT39-03,,GA03FLMB-RIM01-02-CTDMOH071,3,1,2016-10-31T22:37:00,2018-01-12T13:07:00,CGMGA-03FLMB-00003,,CGINS-CTDMOH-12394,-42.49601667,-42.12545,,1500,5165,
+NBP1609,,AT39-03,,GA03FLMB-RIS01-00-SIOENG000,3,1,2016-10-31T22:37:00,2018-01-12T10:56:00,CGMGA-03FLMB-00003,,CGCON-SIOCON-13994,-42.49601667,-42.12545,,30,5165,
+NBP1609,,AT39-03,,GA03FLMB-RIS01-03-DOSTAD000,3,1,2016-10-31T22:37:00,2018-01-12T10:56:00,CGMGA-03FLMB-00003,,CGINS-DOSTAD-00386,-42.49601667,-42.12545,,30,5165,
+NBP1609,,AT39-03,,GA03FLMB-RIS01-04-PHSENF000,3,1,2016-10-31T22:37:00,2018-01-12T10:56:00,CGMGA-03FLMB-00003,,CGINS-PHSENF-P0142,-42.49601667,-42.12545,,30,5165,
+NBP1609,,AT39-03,,GA03FLMB-RIS01-05-FLORTD000,3,1,2016-10-31T22:37:00,2018-01-12T10:56:00,CGMGA-03FLMB-00003,,CGINS-FLORTD-01215,-42.49601667,-42.12545,,30,5165,
```
Comment (snwhite) [inline deployment/GA03FLMB_Deploy.csv:55]: Deployment 3 – Looks good.

File: deployment/GS03FLMA_Deploy.csv [hunk 6]
```diff
@@ -1,55 +1,55 @@
...
+AT26-29,,NBP1511,,GS03FLMA-RIM01-02-CTDMOH049,1,1,2015-02-25T23:55:00,2015-12-18T12:33:00,CGMGS-03FLMA-00001,,CGINS-CTDMOH-12385,-54.10517,-89.65361,,747,4632,
+AT26-29,,NBP1511,,GS03FLMA-RIM01-02-CTDMOH050,1,1,2015-02-25T23:55:00,2015-12-18T12:33:00,CGMGS-03FLMA-00001,,CGINS-CTDMOH-12391,-54.10517,-89.65361,,997,4632,
+AT26-29,,NBP1511,,GS03FLMA-RIM01-02-CTDMOH051,1,1,2015-02-25T23:55:00,2015-12-18T12:33:00,CGMGS-03FLMA-00001,,CGINS-CTDMOH-12390,-54.10517,-89.65361,,1497,4632,
+AT26-29,,NBP1511,,GS03FLMA-RIS01-00-SIOENG000,1,1,2015-02-25T23:55:00,2015-12-18T09:46:00,CGMGS-03FLMA-00001,,CGCON-SIOCON-13987,-54.10517,-89.65361,,27,4632,
+AT26-29,,NBP1511,,GS03FLMA-RIS01-03-DOSTAD000,1,1,2015-02-25T23:55:00,2015-12-18T09:46:00,CGMGS-03FLMA-00001,,CGINS-DOSTAD-00393,-54.10517,-89.65361,,27,4632,
+AT26-29,,NBP1511,,GS03FLMA-RIS01-04-PHSENF000,1,1,2015-02-25T23:55:00,2015-12-18T09:46:00,CGMGS-03FLMA-00001,,CGINS-PHSENF-P0147,-54.10517,-89.65361,,27,4632,
+AT26-29,,NBP1511,,GS03FLMA-RIS01-05-FLORTD000,1,1,2015-02-25T23:55:00,2015-12-18T09:46:00,CGMGS-03FLMA-00001,,CGINS-FLORTD-01003,-54.10517,-89.65361,,27,4632,
```
Comment (snwhite) [inline deployment/GS03FLMA_Deploy.csv:19]: Deployment 1 – Looks good.

File: deployment/GS03FLMA_Deploy.csv [hunk 7]
```diff
@@ -1,55 +1,55 @@
...
+NBP1511,,NBP1610,,GS03FLMA-RIM01-02-CTDMOH049,2,1,2015-12-17T19:42:00,2016-12-06T11:28:00,CGMGS-03FLMA-00002,,CGINS-CTDMOH-13652,-54.12563,-89.55293,,747,4705,
+NBP1511,,NBP1610,,GS03FLMA-RIM01-02-CTDMOH050,2,1,2015-12-17T19:42:00,2016-12-06T11:28:00,CGMGS-03FLMA-00002,,CGINS-CTDMOH-13653,-54.12563,-89.55293,,997,4705,
+NBP1511,,NBP1610,,GS03FLMA-RIM01-02-CTDMOH051,2,1,2015-12-17T19:42:00,2016-12-06T11:28:00,CGMGS-03FLMA-00002,,CGINS-CTDMOH-13655,-54.12563,-89.55293,,1497,4705,
+NBP1511,,NBP1610,,GS03FLMA-RIS01-00-SIOENG000,2,1,2015-12-17T19:42:00,2016-12-06T09:00:00,CGMGS-03FLMA-00002,,CGCON-SIOCON-14877,-54.12563,-89.55293,,27,4705,
+NBP1511,,NBP1610,,GS03FLMA-RIS01-03-DOSTAD000,2,1,2015-12-17T19:42:00,2016-12-06T09:00:00,CGMGS-03FLMA-00002,,CGINS-DOSTAD-00498,-54.12563,-89.55293,,27,4705,
+NBP1511,,NBP1610,,GS03FLMA-RIS01-04-PHSENF000,2,1,2015-12-17T19:42:00,2016-12-06T09:00:00,CGMGS-03FLMA-00002,,CGINS-PHSENF-P0182,-54.12563,-89.55293,,27,4705,
+NBP1511,,NBP1610,,GS03FLMA-RIS01-05-FLORTD000,2,1,2015-12-17T19:42:00,2016-12-06T09:00:00,CGMGS-03FLMA-00002,,CGINS-FLORTD-01321,-54.12563,-89.55293,,27,4705,
```
Comment (snwhite) [inline deployment/GS03FLMA_Deploy.csv:37]: Deployment 2 – Looks good.

File: deployment/GS03FLMA_Deploy.csv [hunk 8]
```diff
@@ -1,55 +1,55 @@
...
+NBP1610,,DY096,,GS03FLMA-RIM01-02-CTDMOH050,3,1,2016-12-01T19:20:00,2018-12-10T10:11:00,CGMGS-03FLMA-00003,,CGINS-CTDMOH-12393,-54.10421167,-89.65421167,,1000,4627,bottom portion of mooring left in an extra year due to weather
+NBP1610,,DY096,,GS03FLMA-RIM01-02-CTDMOH051,3,1,2016-12-01T19:20:00,2018-12-10T10:11:00,CGMGS-03FLMA-00003,,CGINS-CTDMOH-12391,-54.10421167,-89.65421167,,1500,4627,bottom portion of mooring left in an extra year due to weather
+NBP1610,,NBP1709,,GS03FLMA-RIS01-00-SIOENG000,3,1,2016-12-01T19:20:00,2017-12-03T17:19:00,CGMGS-03FLMA-00003,,CGCON-SIOCON-13998,-54.10421167,-89.65421167,,30,4627,
+NBP1610,,NBP1709,,GS03FLMA-RIS01-03-DOSTAD000,3,1,2016-12-01T19:20:00,2017-12-03T17:19:00,CGMGS-03FLMA-00003,,CGINS-DOSTAD-00131,-54.10421167,-89.65421167,,30,4627,
+NBP1610,,NBP1709,,GS03FLMA-RIS01-04-PHSENF000,3,1,2016-12-01T19:20:00,2017-12-03T17:19:00,CGMGS-03FLMA-00003,,CGINS-PHSENF-P0144,-54.10421167,-89.65421167,,30,4627,
+NBP1610,,NBP1709,,GS03FLMA-RIS01-05-FLORTD000,3,1,2016-12-01T19:20:00,2017-12-03T17:19:00,CGMGS-03FLMA-00003,,CGINS-FLORTD-01003,-54.10421167,-89.65421167,,30,4627,
```
Comment (snwhite) [inline deployment/GS03FLMA_Deploy.csv:55]: Deployment 3 – Looks good.

File: deployment/GS03FLMB_Deploy.csv [hunk 9]
```diff
@@ -1,55 +1,55 @@
...
+AT26-29,,NBP1511,,GS03FLMB-RIM01-02-CTDMOH069,1,1,2015-02-23T20:19:00,2015-12-19T18:40:00,CGMGS-03FLMB-00001,,CGINS-CTDMOH-12393,-54.0803,-88.9346,,747,4932,
+AT26-29,,NBP1511,,GS03FLMB-RIM01-02-CTDMOH070,1,1,2015-02-23T20:19:00,2015-12-19T18:40:00,CGMGS-03FLMB-00001,,CGINS-CTDMOH-12392,-54.0803,-88.9346,,997,4932,
+AT26-29,,NBP1511,,GS03FLMB-RIM01-02-CTDMOH071,1,1,2015-02-23T20:19:00,2015-12-19T18:40:00,CGMGS-03FLMB-00001,,CGINS-CTDMOH-12384,-54.0803,-88.9346,,1497,4932,
+AT26-29,,NBP1511,,GS03FLMB-RIS01-00-SIOENG000,1,1,2015-02-23T20:19:00,2015-12-19T16:40:00,CGMGS-03FLMB-00001,,CGCON-SIOCON-13997,-54.0803,-88.9346,,27,4932,
+AT26-29,,NBP1511,,GS03FLMB-RIS01-03-DOSTAD000,1,1,2015-02-23T20:19:00,2015-12-19T16:40:00,CGMGS-03FLMB-00001,,CGINS-DOSTAD-00392,-54.0803,-88.9346,,27,4932,
+AT26-29,,NBP1511,,GS03FLMB-RIS01-04-PHSENF000,1,1,2015-02-23T20:19:00,2015-12-19T16:40:00,CGMGS-03FLMB-00001,,CGINS-PHSENF-P0144,-54.0803,-88.9346,,27,4932,
+AT26-29,,NBP1511,,GS03FLMB-RIS01-05-FLORTD000,1,1,2015-02-23T20:19:00,2015-12-19T16:40:00,CGMGS-03FLMB-00001,,CGINS-FLORTD-01217,-54.0803,-88.9346,,27,4932,
```
Comment (snwhite) [inline deployment/GS03FLMB_Deploy.csv:19]: Deployment 1 – Looks good.

File: deployment/GS03FLMB_Deploy.csv [hunk 10]
```diff
@@ -1,55 +1,55 @@
...
+NBP1511,,NBP1610,,GS03FLMB-RIM01-02-CTDMOH069,2,1,2015-12-16T19:33:00,2016-12-05T14:07:00,CGMGS-03FLMB-00002,,CGINS-CTDMOH-13649,-54.0824,-88.85905,,747,4942,
+NBP1511,,NBP1610,,GS03FLMB-RIM01-02-CTDMOH070,2,1,2015-12-16T19:33:00,2016-12-05T14:07:00,CGMGS-03FLMB-00002,,CGINS-CTDMOH-13650,-54.0824,-88.85905,,997,4942,
+NBP1511,,NBP1610,,GS03FLMB-RIM01-02-CTDMOH071,2,1,2015-12-16T19:33:00,2016-12-05T14:07:00,CGMGS-03FLMB-00002,,CGINS-CTDMOH-13633,-54.0824,-88.85905,,1497,4942,
+NBP1511,,NBP1610,,GS03FLMB-RIS01-00-SIOENG000,2,1,2015-12-16T19:33:00,2016-12-05T11:45:00,CGMGS-03FLMB-00002,,CGCON-SIOCON-14885,-54.0824,-88.85905,,27,4942,
+NBP1511,,NBP1610,,GS03FLMB-RIS01-03-DOSTAD000,2,1,2015-12-16T19:33:00,2016-12-05T11:45:00,CGMGS-03FLMB-00002,,CGINS-DOSTAD-00502,-54.0824,-88.85905,,27,4942,
+NBP1511,,NBP1610,,GS03FLMB-RIS01-04-PHSENF000,2,1,2015-12-16T19:33:00,2016-12-05T11:45:00,CGMGS-03FLMB-00002,,CGINS-PHSENF-P0183,-54.0824,-88.85905,,27,4942,
+NBP1511,,NBP1610,,GS03FLMB-RIS01-05-FLORTD000,2,1,2015-12-16T19:33:00,2016-12-05T11:45:00,CGMGS-03FLMB-00002,,CGINS-FLORTD-01322,-54.0824,-88.85905,,27,4942,
```
Comment (snwhite) [inline deployment/GS03FLMB_Deploy.csv:37]: Deployment 2 – Looks good.

File: deployment/GS03FLMB_Deploy.csv [hunk 11]
```diff
@@ -1,55 +1,55 @@
...
+NBP1610,,DY096,,GS03FLMB-RIM01-02-CTDMOH070,3,1,2016-12-03T19:08:00,2018-12-10T17:57:00,CGMGS-03FLMB-00003,,CGINS-CTDMOH-12385,-54.08030667,-88.93177167,,1000,4928,bottom portion of mooring left in an extra year due to weather
+NBP1610,,DY096,,GS03FLMB-RIM01-02-CTDMOH071,3,1,2016-12-03T19:08:00,2018-12-10T17:57:00,CGMGS-03FLMB-00003,,CGINS-CTDMOH-12384,-54.08030667,-88.93177167,,1500,4928,bottom portion of mooring left in an extra year due to weather
+NBP1610,,NBP1709,,GS03FLMB-RIS01-00-SIOENG000,3,1,2016-12-03T19:08:00,2017-12-03T12:32:00,CGMGS-03FLMB-00003,,CGCON-SIOCON-13997,-54.08030667,-88.93177167,,30,4928,
+NBP1610,,NBP1709,,GS03FLMB-RIS01-03-DOSTAD000,3,1,2016-12-03T19:08:00,2017-12-03T12:32:00,CGMGS-03FLMB-00003,,CGINS-DOSTAD-00412,-54.08030667,-88.93177167,,30,4928,
+NBP1610,,NBP1709,,GS03FLMB-RIS01-04-PHSENF000,3,1,2016-12-03T19:08:00,2017-12-03T12:32:00,CGMGS-03FLMB-00003,,CGINS-PHSENF-P0141,-54.08030667,-88.93177167,,30,4928,
+NBP1610,,NBP1709,,GS03FLMB-RIS01-05-FLORTD000,3,1,2016-12-03T19:08:00,2017-12-03T12:32:00,CGMGS-03FLMB-00003,,CGINS-FLORTD-01214,-54.08030667,-88.93177167,,30,4928,
```
Comment (snwhite) [inline deployment/GS03FLMB_Deploy.csv:55]: Deployment 3 – Looks good.

File: deployment/GA03FLMA_Deploy.csv (hunk 1 above)
Comment (dbogorff) [inline deployment/GA03FLMA_Deploy.csv:19]: Dep 1 looks good!

File: deployment/GA03FLMA_Deploy.csv (hunk 2 above)
Comment (dbogorff) [inline deployment/GA03FLMA_Deploy.csv:37]: Dep 2 looks good!

File: deployment/GA03FLMA_Deploy.csv (hunk 3 above)
Comment (dbogorff) [inline deployment/GA03FLMA_Deploy.csv:55]: Dep 3 Looks good!

File: deployment/GA03FLMB_Deploy.csv (hunk 4 above)
Comment (dbogorff) [inline deployment/GA03FLMB_Deploy.csv:37]: Dep 1/2 looks good!

File: deployment/GA03FLMB_Deploy.csv (hunk 5 above)
Comment (dbogorff) [inline deployment/GA03FLMB_Deploy.csv:55]: Dep 3 looks good!

File: deployment/GS03FLMA_Deploy.csv (hunk 6 above)
Comment (dbogorff) [inline deployment/GS03FLMA_Deploy.csv:19]: Dep 1 looks good!

File: deployment/GS03FLMA_Deploy.csv (hunk 7 above)
Comment (dbogorff) [inline deployment/GS03FLMA_Deploy.csv:37]: Dep 2 looks good!

File: deployment/GS03FLMA_Deploy.csv (hunk 8 above)
Comment (dbogorff) [inline deployment/GS03FLMA_Deploy.csv:55]: Dep 3 looks good!

File: deployment/GS03FLMB_Deploy.csv (hunk 9 above)
Comment (dbogorff) [inline deployment/GS03FLMB_Deploy.csv:19]: Dep 1 looks good!

File: deployment/GS03FLMB_Deploy.csv (hunk 10 above)
Comment (dbogorff) [inline deployment/GS03FLMB_Deploy.csv:37]: Dep 2 looks good!

File: deployment/GS03FLMB_Deploy.csv (hunk 11 above)
Comment (dbogorff) [inline deployment/GS03FLMB_Deploy.csv:55]: Dep 3 looks good!

**Pull request merged**